building/build-debs/homeworld-keysystem/src/keycommon/knc/knc.go

```diff
@@ -0,0 +1,58 @@
+package knc
+
+import (
+	"fmt"
+	"errors"
+	"os/exec"
+	"encoding/json"
+	"keycommon/reqtarget"
+)
+
+type KncServer struct {
+	Hostname string
+}
+
+func (k KncServer) kncRequest(data []byte) ([]byte, error) {
+	cmd := exec.Command("/usr/bin/knc", fmt.Sprintf("host@%s", k.Hostname), "20575")
```

Comment:
The service name is always `host`?

Reply:
Yes. Because our keytabs are for `host/egg-sandwich.mit.edu` and such.